cran-comments.md

```diff
@@ -1,12 +1,12 @@
 ## Test environments
```

Comment:
The tidyverse teams dropped the "Test environments" section from their CRAN comments template because they felt it wasn't used by CRAN anyway https://github.com/r-lib/usethis/issues/1365